app/mattermost_managed/mattermost-managed.android.js

```diff
@@ -66,5 +66,6 @@ export default {
 
         return JailMonkey.trustFall();
     },
+    supportsFaceId: async () => false,
```

Comment:
0/5 for now, but probably there are some Android devices that support this feature

Reply:
Face ID is trademarked by Apple, so they couldn't technically support "Face ID" 🙃